src/components/Deposit/Deposit.tsx

```diff
@@ -41,12 +41,13 @@ import { toFloat } from '../../utils/string';
 import { makeTarget, minSufficientValueForOrder } from '../../utils/ammMath';
 import { calculateTotalFee } from '../../utils/transactions';
 import { renderPoolPrice } from '../../utils/price';
+import { DexFeeDefault } from '../../constants/settings';
```

Comment:
use UPPER_SNAKE_CASE to define global constants.
Move it to the `erg.js` file. This file is a storage for Ergo blockchain consts

Reply:
@yasha-black `DexFeeDefault` is not a constant.